aspnetcore/performance/caching/memory/sample/WebCache/Startup.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+public class Startup
+{
+    public void ConfigureServices(IServiceCollection services)
+    {
+        // Requires NuGet package "Microsoft.Extensions.Caching.Memory"
```

Comment:
As per previous discussion, let's remove this comment then. It's being called out in the docs explicitly